lib/workers/repository/update/branch/bump-versions.ts

```diff
@@ -57,25 +57,48 @@ async function bumpVersion(
 ): Promise<void> {
   const rawBumpType = config.bumpType ?? 'patch';
 
+  // all log messages should be prefixed with this string to allow logLevelRemapping
```

Comment:
```suggestion
  // all log messages should be prefixed with this string to facilitate easier logLevelRemapping
```

Reply:
🤷

Reply:
Seemed better as `logLevelRemapping` is still possible :) ....